.github/workflows/test.yml

```diff
@@ -220,14 +221,21 @@ jobs:
           - target: s390x-unknown-linux-gnu
             arch: s390x
             abi: cp310-cp310
+          # PyPy
+          - target: aarch64-unknown-linux-gnu
+            arch: aarch64
+            abi: pp37-pypy37_pp73
+          - target: aarch64-unknown-linux-gnu
+            arch: aarch64
+            abi: pp38-pypy38_pp73
     steps:
       - uses: actions/checkout@v2
       - name: Build Wheels
         run: |
           echo 'curl -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain stable --profile minimal
           source ~/.cargo/env
           rustup target add ${{ matrix.platform.target }}
-          export PYO3_CROSS_LIB_DIR=/opt/python/${{ matrix.platform.abi }}/lib
+          export PYO3_CROSS_LIB_DIR=/opt/python/${{ matrix.platform.abi }}
```

Comment:
@davidhewitt For #681 I'd like to change pyo3 to support `PYO3_CROSS_LIB_DIR` without the `lib` part, what do you think?

Reply:
cc @aganders3 in case you are interested in for https://github.com/PyO3/pyo3/pull/1996